task/buildpacks/0.2/README.md

```diff
@@ -0,0 +1,127 @@
+
+See also [`buildpacks-phases`](../buildpacks-phases) for the deconstructed version of this task, which runs each of the [lifecycle phases](https://buildpacks.io/docs/concepts/components/lifecycle/#phases) individually (this task uses the [creator binary](https://github.com/buildpacks/spec/blob/platform/0.3/platform.md#operations), which coordinates and runs all of the phases).
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/buildpacks/0.2/buildpacks.yaml
+```
+
+> **NOTE:** This task is currently only compatible with Tekton **v0.11.0** and above, and CNB Platform API 0.3 (lifecycle v0.7.0 and above). For previous Platform API versions, [see below](#previous-platform-api-versions).
+
+## Parameters
+
+* **`BUILDER_IMAGE`**: The image on which builds will run. (must include lifecycle and compatible buildpacks; _required_)
+
+* **`CACHE`**: The name of the persistent app cache volume. (_default:_ an empty directory -- effectively no cache)
+
+* **`CACHE_IMAGE`**: The name of the persistent app cache image. (_default:_ no cache image)
+
+* **`PLATFORM_DIR`**: A directory containing platform provided configuration, such as environment variables.
+  Files of the format `/platform/env/MY_VAR` with content `my-value` will be translated by the lifecycle into environment variables provided to buildpacks. For more information, see the [buildpacks spec](https://github.com/buildpacks/spec/blob/master/buildpack.md#provided-by-the-platform). (_default:_ an empty directory)
+
+* **`USER_ID`**: The user ID of the builder image user, as a string value. (_default:_ `"1000"`)
+
+* **`GROUP_ID`**: The group ID of the builder image user, as a string value. (_default:_ `"1000"`)
+
+* **`PROCESS_TYPE`**: The default process type to set on the image. (_default:_ `"web"`)
+
+* **`SKIP_RESTORE`**: Do not write layer metadata or restore cached layers. (clear cache between each run) (_default:_ `"false"`)
+
+* **`RUN_IMAGE`**: Reference to a run image to use. (_default:_ run image of the builder)
+
+* **`SOURCE_SUBPATH`**: A subpath within the `source` input where the source to build is located. (_default:_ `""`)
+
+### Outputs
+
+* **`image`**: An `image`-type `PipelineResource` specifying the image that should
+  be built.
+
+## Workspaces
+
+The `source` workspace holds the source to build. See `SOURCE_SUBPATH` above if source is located within a subpath of this input.
+
+## Usage
+
+This `TaskRun` will use the `buildpacks` task to build the source code, then publish a container image.
+
+```
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: example-run
+spec:
+  taskRef:
+    name: buildpacks
+  podTemplate:
+    volumes:
+# Uncomment the lines below to use an existing cache
```

Comment:
We could use optional workspace here, isnt it ?

Reply:
We definitely could.